plugins/gimp/file-jxl-save.cc

```diff
@@ -10,122 +10,368 @@
 #undef MIN
 #undef CLAMP
 
-#include "lib/jxl/alpha.h"
+#include <glib/gi18n.h>
+#include <libgimp/gimp.h>
+#include <libgimp/gimpui.h>
+//#include <glib/gstdio.h>
```

Comment:
This seems to be a leftover.

Reply:
`glib/gi18n.h` is used for the `_(string)` wrappers.  Not sure whether to keep them or not.

I also don't know whether to put `libgimp/gimp*` in `file-jxl-save.h`.

`glib/gstdio.h` is commented out because I didn't know whether I'd need it later.  So far, it looks like I won't, so I'll probably just remove it.